### ISSUE TITLE
Signing pre-generated transaction envelopes

### ISSUE THREAD

**Issue by sinaghaffari** [johansten/stargazer#61]
It would be great if Stargazer let users enter the XDR of an existing, unsigned transaction and let them sign it with their accounts.
Perhaps allowing users to scan a QR code containing said XDR encoded envelopes to sign?

That way, I can generate a transaction and display a QR code to the user that they can scan, review and sign securely.

**Comment by johansten** [comment]
Are you thinking in terms of offline signing? Generate the transaction on one device, sign on another?

**Comment by sinaghaffari** [comment]
Yes, precisely! The user can see the contents of the transaction and choose to sign and submit it.

**Comment by sinaghaffari** [comment]
My proposal is a QR code that contain two things.

1) An unsigned envelope to be signed by the user.
2) A callback, for the transaction to be sent to after it has been signed. This way the user isn't directly responsible for submitting the transaction (in case there are multiple signatures needed)

```json
{
    "stellar": {
        "unsigned_tx": "AAAA.....==",
        "callback": "https://example.org/authorize"
    } 
}
```

**Comment by prodoxx** [comment]
I'm also interested in this solution. Did you figure it out @sinaghaffari ?
